source/features/search-releases.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+
+import features from '../libs/features';
+import {getOwnerAndRepo} from '../libs/utils';
+import {octoface} from '../libs/icons';
+
+async function init() {
+	if (!hasAnyReleases()) {
+		return false;
+	}
+
+	addSelectMenu();
+	registerEventListener();
+}
+
+const hasAnyReleases = () => select.exists('.release-entry') || select.exists('.release');
+
+const addSelectMenu = () => {
+	const container = select('.subnav');
+	const {ownerName, repoName} = getOwnerAndRepo();
+	const selectedTag = getSelectedTagFromUrl();
+
+	container.append(
+		<div class="float-right d-flex flex-shrink-0 flex-items-center mb-3">
+			<details class="details-reset details-overlay select-menu branch-select-menu">
+				<summary class="btn btn-sm select-menu-button css-truncate" data-hotkey="w" title="Filter releases" aria-haspopup="menu">
+					<i>Filter:</i>&nbsp;
+					<span class="css-truncate-target">{selectedTag || 'Releases'}</span>&nbsp;
+				</summary>
+				<details-menu
+					class="rgh-release-details select-menu-modal position-absolute"
+					src={`/${ownerName}/${repoName}/ref-list/master?source_action=disambiguate&source_controller=files`}
+					preload
+					role="menu"
+					style={{zIndex: 99}}
+				>
+					<include-fragment class="select-menu-loading-overlay anim-pulse">
+						{octoface()}
+					</include-fragment>
+				</details-menu>
+			</details>
+		</div>
+	);
+};
+
+const getSelectedTagFromUrl = () => {
+	const [tag = null] = location.href.match(/(?<=\/releases\/tag\/)([^/]+)/) || [];
+
+	return tag;
+};
+
+const registerEventListener = () => {
+	const includeFragment = select('.rgh-release-details > include-fragment');
+
+	return includeFragment ? includeFragment.addEventListener('loadend', onFragmentLoaded) : null;
+};
+
+const onFragmentLoaded = () => {
+	const textInput = select('.rgh-release-details > tab-container > .select-menu-filters > .select-menu-text-filter > input');
+
+	textInput.setAttribute('aria-label', 'Type to filter releases');
+	textInput.setAttribute('placeholder', 'Type to filter releases...');
+
+	select('.rgh-release-details > tab-container > .select-menu-list').remove();
+
+	select('.rgh-release-details > tab-container > .select-menu-list').removeAttribute('hidden');
```

Comment:
I think this selector is wrong, the same element is being `remove`d

Reply:
My bad. It's ambiguous because the same selector is present on two elements. How about doing?

```
const menu = select.all('tab-container > .select-menu-list');

menu[0].remove();
menu[1].removeAttribute('hidden');
```

Reply:
Gotcha. Yeah I think that would be better, also because it becomes one `select`

However I'd use destructuring:

```js
////// use the right names, I don't know what these are
const [nameThis, nameThat] = select.all('tab-container > .select-menu-list');
nameThis.remove();
nameThat.removeAttribute('hidden');
```

Reply:
Cool. Implemented. 👍

Reply:
Ah, I see... maybe we can just use:

```js
 	select('.rgh-release-details tab-container').remove(); 😅
```

Reply:
Yes, You got it right. We cannot. Because doing so will remove the lists, too.